<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProstaFuseResults</class>
 <widget class="QDialog" name="ProstaFuseResults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>616</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Registration Results</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="m_ButtonHorizontalLayout_6">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Slice: </string>
           </property>
           <property name="indent">
            <number>13</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="m_MovedImageSlider">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="m_ButtonHorizontalLayout_8">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Fixed Image Intensity</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <property name="margin">
             <number>1</number>
            </property>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>8</number>
              </property>
              <property name="topMargin">
               <number>6</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item row="1" column="0">
               <widget class="QLabel" name="label_10">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>    Level:   </string>
                </property>
                <property name="indent">
                 <number>4</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_9">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>    Window:   </string>
                </property>
                <property name="indent">
                 <number>4</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSlider" name="m_FixedLevelSlider">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBelow</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSlider" name="m_FixedWindowSlider">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBelow</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Moved Image Intensity</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <property name="margin">
             <number>1</number>
            </property>
            <item>
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="leftMargin">
               <number>8</number>
              </property>
              <property name="topMargin">
               <number>6</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item row="1" column="0">
               <widget class="QLabel" name="label_15">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>    Level:   </string>
                </property>
                <property name="indent">
                 <number>4</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_16">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>    Window:   </string>
                </property>
                <property name="indent">
                 <number>4</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSlider" name="m_MovedLevelSlider">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBelow</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSlider" name="m_MovedWindowSlider">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBelow</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QVTKWidget" name="m_ImageWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="m_ButtonHorizontalLayout">
         <item>
          <widget class="QPushButton" name="m_SaveMovedImage">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(63, 127, 101);
color: rgb(255, 255, 255)</string>
           </property>
           <property name="text">
            <string>&amp;Save Moved Image</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_OptionsButton">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(63, 127, 101);
color: rgb(255, 255, 255)</string>
           </property>
           <property name="text">
            <string>&amp;Options</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="m_CloseButton">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(53, 53, 53);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>&amp;Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_CloseButton</sender>
   <signal>clicked()</signal>
   <receiver>ProstaFuseResults</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>656</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_OptionsButton</sender>
   <signal>clicked()</signal>
   <receiver>ProstaFuseResults</receiver>
   <slot>SlotOptions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_MovedImageSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ProstaFuseResults</receiver>
   <slot>SlotChangeImageSlice(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_SaveMovedImage</sender>
   <signal>clicked()</signal>
   <receiver>ProstaFuseResults</receiver>
   <slot>SlotSaveMovedImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_FixedLevelSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ProstaFuseResults</receiver>
   <slot>SlotChangeFixedLevel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_FixedWindowSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ProstaFuseResults</receiver>
   <slot>SlotChangeFixedWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_MovedLevelSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ProstaFuseResults</receiver>
   <slot>SlotChangeMovedLevel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_MovedWindowSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ProstaFuseResults</receiver>
   <slot>SlotChangeMovedWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SlotOptions()</slot>
  <slot>SlotSaveMovedImage()</slot>
  <slot>SlotChangeImageSlice(int)</slot>
  <slot>SlotChangeMovedLevel()</slot>
  <slot>SlotChangeFixedLevel()</slot>
  <slot>SlotChangeMovedWindow()</slot>
  <slot>SlotChangeFixedWindow()</slot>
 </slots>
</ui>
